src/deadline/client/cli/_groups/manifest_group.py

```diff
@@ -147,13 +156,43 @@ def manifest_diff(
     include: List[str],
     exclude: List[str],
     include_exclude_config: str,
+    force_rehash: bool,
     json: bool,
     **args,
 ):
     """
     Check file differences between a directory and specified manifest.
     """
-    raise NotImplementedError("This CLI is being implemented.")
+    logger: ClickLogger = ClickLogger(is_json=json)
+    if not os.path.isfile(manifest):
+        raise NonValidInputError(f"Specified manifest file {manifest} does not exist. ")
+
+    if not os.path.isdir(root):
+        raise NonValidInputError(f"Specified root directory {root} does not exist. ")
+
+    # Perform the diff.
+    differences = _manifest_diff(
+        manifest=manifest,
+        root=root,
+        include=include,
+        exclude=exclude,
+        include_exclude_config=include_exclude_config,
+        force_rehash=force_rehash,
+        logger=logger,
+    )
+
+    # Print results to console.
+    if json:
+        logger.json(dataclasses.asdict(differences), indent=4)
```

Comment:
What do you mean? logger.json takes in a dictionary so it prints out JSON.

Reply:
The `json` function in clicklogger already check `if json`, this line doesn't need to be under the json check again.

Reply:
Oh! Thats what you meant. Yeah I recognize that in this case, but given there's a line for the pretty print option it was easier to already early escape it here. 

The pretty print does alot more un-necessary work so I wanted to avoid running that code for no reason.